.github/workflows/windows-ci.yml

```diff
@@ -7,14 +7,14 @@ jobs:
     strategy:
       matrix:
         platform:
+          - { image: windows-2022, generator: Visual Studio 17 2022, toolset: v143, cmake-arch: x64, arch: x64, python: "3.13", unity: OFF, str: windows-x64-v143 }
```

Comment:
Do we want `unity: OFF` here? I guess same question for the ones further down...

Is the goal of having the option just to make sure that don't introduce inadvertent compile-time issues between unified and non-unified sources?